Assets/SEE/UI/PropertyDialog/NodePropertyDialog.cs

```diff
@@ -1,6 +1,11 @@
 ﻿using SEE.Controls;
+using SEE.Controls.Actions; /// Reference in comment.
```

Comment:
This using is shown as unused because the reference is 'only' used in a comment. If the unnecessary usings were removed via VS, this using would be removed, and the reference in the comment would no longer work. @tinxx suggested that we add a comment to such usings.